dremio_client/model/endpoints.py

```diff
@@ -654,3 +654,13 @@ def graph(token, base_url, cid=None, ssl_verify=True):
     if cid is None:
         raise TypeError("resource id can't be None for a graph call")
     return _get(base_url + "/api/v3/catalog/{}/graph".format(cid), token, ssl_verify=ssl_verify)
+
+def delete_personal_access_token_without_userid(token, base_url, tid=None, ssl_verify=True):
```

Comment:
we can combine this with teh existing method. 
The pseudo-code below is an example:
```
def delete_personal_access_token(token, base_url, uid=None, tid=None, ssl_verify=True):
    has_user = "user/{}/".format(uid) if uid else ""
    url = base_url + "api/v3/{}token{}".format(has_user, ("/"+tid) if tid else "")
    return _delete(url, token, ssl_verify=ssl_verify)
```

Reply:
@rymurr Okay I will do it that way. I made new functions, because I felt some negative tests that give None as uid might fail.